test/Microsoft.Azure.Devices.Edge.Test.Common/certs/Certificates.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Certs
+{
+    public class Certificates
```

Comment:
Previously we had "edge certificates" and "leaf certificates". Edge certs were really CA certs, and leaf certs were really "identity" (aka end-entity) certs. With your new test, this distinction isn't enough because now we have edge CA certs and edge identity certs (as well as leaf identity certs). Maybe the names for the two cert classes should be `CaCertificate` and `IdentityCertificate`.

`CertificateAuthority` would return these certificate objects from `GenerateCaCertificate()` and `GenerateIdentityCertificate()` respectively.

Reply:
Oh I see what you mean. I'll work on this suggestion.

Reply:
` Certificates ` -> `IdCertificates`
`EdgeCertificate` -> `CaCertificates`